cardano-lib/private/conway-genesis.json

```diff
@@ -32,7 +33,57 @@
+      "keyHash-5560027afff0b1e8c7223a229214203aa0c336ed0932b52fb871ec06": 50000,
+      "keyHash-597abafee698679c9a045b0cf95cac9223e138f23008b91ec0bf3cbe": 50000,
+      "keyHash-5e39fe0f94f372a7079082ce504578a0638a0fc1658a1ecf613b1c4a": 50000,
+      "keyHash-631c7df2de2aabf134b2c42fc89a845df12abfb2b4840d49602f3e4e": 50000,
+      "keyHash-656f0ffbee95ba476ff1e6989b6a40c9a0a3cf953ea60ab924afeeab": 50000,
+      "keyHash-7ba5b199bc44c68a84c66ab933ea4c6e9b6794b1c25281d9505578d0": 50000,
+      "keyHash-8722ba9e8a6d65fc08a797aa157d43775f7c7c76fdf2f41e0fd31eba": 50000,
+      "keyHash-a041a9a722ea330631e20ea821ad38d54c2b5b14c07d3a6b6704f8cb": 50000,
+      "keyHash-a12645c93421187d095c879b76312da65b1c2b979b75eb677c7d721c": 50000,
+      "keyHash-b5236e33c18c0504d3c3ab1dddfee72a07e40dac902855938a30a5dd": 50000,
+      "keyHash-b6d72cbd42ff1b24f78e64ed8b2a36ce414c1cf21f71269ef59d371b": 50000,
+      "keyHash-b7fb7439fce53deb1167ea550188a761fb3fe310d572d490bae75799": 50000,
+      "keyHash-b87a52453dde6410e0de49dd7afbbfb54504e6abf2437ef3916bc011": 50000,
+      "keyHash-b91a9d66011482bcb723e11bd932b335fc19b4dc202f0a2883f77441": 50000,
+      "keyHash-c1a6ef82a7e85b3c28be5aa9d0b4bc6cf8f3a99e01178d632ef44f3b": 50000,
+      "keyHash-c1c0303d21350fd4f0d233d51cad4c082d7af6cbe2eb4217ded6896c": 50000,
+      "keyHash-cf74c9187f434fd5b6ef46f241be5385c762d08606c9b02a14e13ecc": 50000,
+      "keyHash-cfdd096d6c4dfa5d3746dc4e57809569921fa8ea6e9bd35fe1216d90": 50000,
+      "keyHash-d3894963c57fbfd08380ab02a47ea874b84108a57cbecd867dce012f": 50000,
+      "keyHash-d47e346379563e2ee7586c18eb9a5e69cefe6638406b3ec2db84002e": 50000,
+      "keyHash-d5a29623d8205f73adafc536d496072d26f50f1e3eb5cfc2771192bf": 50000,
+      "keyHash-daf1c569670cd5f652eda0a3da569a2504b11e34aed847d1f15c3368": 50000,
+      "keyHash-dbdfe1f5c4417e5ebbf5a2d4c1fa2bbd9c8607c54a5f58ea690af079": 50000,
+      "keyHash-e4abbc9c02ca4f928fff87a966c82b9a0669039ef34039ffb50adeb8": 50000,
+      "keyHash-e9408514b3052bd795221b83b96cd0caaecbcb4a2498f518c3ae9afa": 50000,
+      "keyHash-ebb7f6492d81674a003d6aee6bd996f2abf69f235a5d38f2e56480ab": 50000,
+      "keyHash-ec817e7e6a255f9cee08dfe2ea8889e978d7d5b82ac84c890297e5f9": 50000,
+      "keyHash-f0654c6a33a88b6f5b6409eb2bece233280c02d6ed9f8fb9889a0658": 50000,
+      "keyHash-f10a298d2b5eca9bb8c7b13048f56ad609dc590b7227dc7fb014a68a": 50000,
+      "keyHash-f15eddcc5fc03363a10190c28967f1ebe8137a590b1303de014d3741": 50000,
+      "keyHash-f225def146b47cc5fcc409e0321a2463a6bc03c54fb544542f1f4e98": 50000,
+      "keyHash-f398a0e594625784f610fdb259d0d60174f747f051e8fa6147e6473d": 50000,
+      "keyHash-f631cbed22011910734df143167a59d4dabadac67d95194238763bbf": 50000,
+      "keyHash-fa87c7a1a9b28b478d798500ca2b263e423eb4c25507a153d11b294e": 50000,
+      "keyHash-feac670c85cab8ec936fdaa255bd0d1ea3adf717043c5f2727736f15": 50000,
+      "keyHash-ff592f3b0aef7854ee5296ca9c20175a3e96bdf9da9d127e3c5828d5": 50000
     },
-    "quorum": 0
```

Comment:
This appears to cause a problem with db-sync being able to parse the conway genesis file as it expects this quorum key to be present.